src/Raven.Server/RavenServerStartup.cs

```diff
@@ -154,6 +156,7 @@ public static bool IsHtmlAcceptable(HttpContext context)
             {
                 context.Response.StatusCode = (int)HttpStatusCode.OK;
                 context.Response.Headers["Content-Type"] = "application/json; charset=utf-8";
+                context.Response.Headers[Constants.Headers.ServerVersion] = RavenVersionAttribute.Instance.Build;
```

Comment:
Not sure if this is a good idea, for dev build, that will always be 13, for example.
Means that we can't inter-operate between dev and official builds.
Better to use a fixed version, such as `4.0`, etc

Reply:
The dev build for 4.0 is 40 and for 4.1 is 41.